### ISSUE TITLE
49Mhz remote control.

### ISSUE THREAD

**Issue by sandmix** [probonopd/Lifestyle#8]
I was looking for a long time the way how to realize transmition on this frequencies as I know rasberry uses PWM for this but as I understood for ESP PWM isn't enought. 
With you solution the signal is mirrored and has a lot of noice and also is weak ?

what about Si4012 the circuit looks very simple and the signal is clean and powerfull.

**Comment by probonopd** [comment]
Wow, didn't know about this device. Sounds interesting:

> The Si4012 is a fully-integrated, crystal-less, sub-GHz CMOS RF transmitter offering industry-leading RF performance, high integration, flexibility, low BOM, small board area, and ease of design.
The device is designed to operate with any host MCU via a serial interface while optimized for battery-powered applications. The Si4012 operates from voltages ranging from 1.8 to 3.6 V and offers an ultra-low standby current consumption of less than 10 nA.
The embedded power amplifier can be programmed to supply from –13 dBm up to +10 dBm, while the patented automatic antenna tuning circuit ensures that the resonant frequency and impedance matching between the PA output and the connected antenna are configured for optimum transmit efficiency and low harmonic content.

There is even an Arduino library for it:
https://github.com/mrmx/Arduino-SI4012

But I don't know yet whether/how it can send the signals we need to send.

**Comment by sandmix** [comment]
It will need some work but I think it's realizable and the circuit looks really simpel:
![image](https://user-images.githubusercontent.com/7332661/116921448-fdfc4a00-ac21-11eb-8214-1e184af3cdf5.png)

**Comment by probonopd** [comment]
I am not so much worried about the "circuit" but more about how to send the OOK codes we need.

**Comment by sandmix** [comment]
Yep, it's not straightforward maybe AD9850 example can help I can see that it is capable but to find out how to do it, this is different story. I'll order couple, will play when I'll have some free time. I would even use AD985 but I think price is to high for it.
Are you still using 8266 for this ?

**Comment by sandmix** [comment]
this one looks also promissing https://github.com/jakeson21/Arduino-LTC6904-Library
https://digibird1.wordpress.com/arduino-controlled-frequency-generator-1khz-68-mhz/
